hunting-buddy.lic

```diff
@@ -20,6 +20,7 @@ class HuntingBuddy
     @escort_zones = data.escort_zones
     @hunting_zones = data.hunting_zones
     @stop_on_low_threshold = @settings.stop_on_low_threshold
+    @stop_on_threshold = @settings.stop_on_threshold || 32
```

Comment:
If it's in `base.yaml`, it doesn't have to be specified here.

Reply:
I can remove it, figured it would be good as a failsafe incase someone on dev mode pulls the updated hunting-buddy.